client/flutter/assets/content_bundles/protect_yourself.en.yaml

```diff
@@ -14,7 +14,7 @@ contents:
         <b>Cover your mouth and nose</b> with your bent elbow or tissue when you cough or sneeze.
     - image_name: protect-distance
       body_html: |
-        <b>Stay more than</b> 1 meter (3 feet) away from a person who is sick.
+        <b>Distance at least a 1 meter</b> (3 feet) with others and an even greater distance when indoors.
```

Comment:
Can you remove the extra "a" in "a 1 meter"

Reply:
Good catch. That shouldn't be there.